libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -106,12 +106,12 @@ pair<string, string> IRGenerator::run(
 	asmStack.optimize();
 
 	string warning =
-		"/*******************************************************\n"
-		" *                       WARNING                       *\n"
-		" *  Solidity to Yul compilation is still EXPERIMENTAL  *\n"
-		" *       It can result in LOSS OF FUNDS or worse       *\n"
-		" *                !USE AT YOUR OWN RISK!               *\n"
-		" *******************************************************/\n\n";
+		"/*****************************************************|\n"
```

Comment:
```suggestion
		"/*====================\n"
```
What is important is that it does not start with `/**`

Reply:
Please fix this.